04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,12 @@
 #write your code here
+def translate (word)
+    vowels = %w{a e i o u}
+    word.gsub(/(\A|\s)\w+/) do |str|
+            str.strip!
+        while not vowels.include? str[0] or (str[0] == 'u' and str[-1] == 'q')
+            str += str[0]
+            str = str[1..-1]
+        end
+        str  = ' ' + str + 'ay'
```

Comment:
You could use string interpolation here and it would work the same: `str = " #{str}ay"`